### ISSUE TITLE
Fix issues link on Accessibility page

### ISSUE THREAD

**Issue by mattstein** [craftcms/spoke-and-chain#23]
### Description

The Accessibility page links to https://github.com/pixelandtonic/spoke-and-chain/issues and should be updated to https://github.com/craftcms/spoke-and-chain/issues.

**Comment by mattstein** [comment]
Fixed in bbf9cc533614e829853828361ecc274b5ae98e43.
